src/routes/posts.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express')
+
+const db = require('../db')
+
+const router = express.Router()
+
+const dateTime = new Date().toJSON()
+
+// CREATE AN ARTICLE
+
+const createPosts = async (req, res, next) => {
+    try {
+        const { id, userId, title, image, content, published, createdAt } =
+            req.body
+        await db.query(
+            'INSERT INTO posts (id , "userId", title , image , content , published , "createdAt" ) VALUES ($1 , $2 , $3 , $4 , $5 ,$6 , $7) RETURNING *',
```

Comment:
@beblicarl, from the postgres setup migration added, the `createdAt` should be populated by default when the document is inserted.

https://github.com/tbf-challenge/teamwork-be/blob/b4a4f5817504c2590f509112e27514c240e2fc14/migrations/1660111562812_db-init.js#L9 

That means you will use the result after the insert to get the fields and then return that as the API response

Not tested at my end so can tell the response that comes from the insert but hopefully it returns the inserted document

As always, ask me anything that you want clarification on

Reply:
so in effect, what i'm saying is you don't have to manually insert the `createdAt` yourself. Let the database do that

Reply:
@beblicarl just saw we're also inserting the id field from the request. that shouldn't be the case

 yea so same for the `id` field especially. It should be assigned from the database and not passed on from the frontend or anyone calling the endpoint

Reply:
> @beblicarl, from the postgres setup migration added, the `createdAt` should be populated by default when the document is inserted.
> 
> https://github.com/tbf-challenge/teamwork-be/blob/b4a4f5817504c2590f509112e27514c240e2fc14/migrations/1660111562812_db-init.js#L9
> 
> That means you will use the result after the insert to get the fields and then return that as the API response
> 
> Not tested at my end so can 't ell the response that comes from the insert but hopefully it returns the inserted document
> 
> As always, ask me anything that you want clarification on

okay, I will work on this as well

Reply:
> so in effect, what i'm saying is you don't have to manually insert the `createdAt` yourself. Let the database do that

I will work on this as well

Reply:
I am done with both the id and createdAt properties @jocrah

Reply:
okay cool, taking a look